webapp/app/components/app.js

```diff
@@ -0,0 +1,47 @@
+import { Application } from 'pixi.js';
+
+import { Field } from './field';
+import { Background } from './background';
+
+
+export class App {
+  constructor() {
+    this.app = null;
+    this.field = null;
+
+    this.crateApp();
+    this.addListeners();
+  }
+
+  crateApp() {
+    this.app = new Application({
+      width: window.innerWidth,
+      height: window.innerHeight,
+      antialias: true,
+      transparent: false,
+      resolution: 1,
```

Comment:
autoResize: true?

Reply:
Co to powinno zrobić? Zgodnie z tym co piszą tutaj: http://pixijs.download/release/docs/PIXI.Application.html to autoResize nie jest propem klasy Application tylko renderera. Dodawałem tutaj i tutaj i nie widziałem zbytnio różnicy.

Reply:
sorki, moj blad :)